### PR TITLE
qemu 2.2.1

### PR DESCRIPTION
**Pull request by DomT4**
Version bump, added optional dep, slight re-jig of the install block for readability, added test.

### CODE REVIEW

**Comment by DomT4** [comment]
@mikemcquaid Let me know how you feel about the test here. It's a little different to the way we usually handle tests, but ` qemu ` is a bit of a nightmare to test in any reasonably simple way that *isn't* just ` blah --version `.

File: Library/Formula/qemu.rb [hunk 1]
```diff
@@ -31,11 +32,27 @@ def install
...
     system "./configure", *args
     system "make", "V=1", "install"
   end
+
+  test do
+    expected_checksum = "4b4c2dce4c055f0a2adb93d571987a3d40c96c6cbfd9244d19b9708ce5aea454"
+    expected_info = /file format: raw/
+
+    # 3.2MB working disc-image file hosted on upstream's servers for people to use to test qemu functionality.
+    # http://wiki.qemu.org/Testing
```
Comment (mikemcquaid) [inline Library/Formula/qemu.rb]: If we want to do this would be good to use a `resource`

Comment (mikemcquaid) [inline Library/Formula/qemu.rb]: (then we'll get the checksum for free)

File: Library/Formula/qemu.rb [hunk 2]
```diff
@@ -31,11 +32,27 @@ def install
...
     args << (build.with?("gtk+") ? "--enable-gtk" : "--disable-gtk")
-    ENV["LIBTOOL"] = "glibtool"
+    args << (build.with?("libssh2") ? "--enable-libssh2" : "--disable-libssh2")
+
     system "./configure", *args
     system "make", "V=1", "install"
   end
+
+  test do
+    expected_checksum = "4b4c2dce4c055f0a2adb93d571987a3d40c96c6cbfd9244d19b9708ce5aea454"
```
Comment (DomT4) [inline Library/Formula/qemu.rb]: We can safely assume this won't change. The last change to this tarball was at the start of 2006.

File: Library/Formula/qemu.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/qemu.rb]: Do we want to keep the ` resource ` around permanently in ` libexec ` or do we just want to stage it in the ` testpath `?

Comment (mikemcquaid) [inline Library/Formula/qemu.rb]: Just `testpath`, thanks!

Comment (DomT4) [inline Library/Formula/qemu.rb]: Done :+1:

**Comment by mikemcquaid** [comment]
Thanks @DomT4!

**Pull request closed without merging**